<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJob</class>
 <widget class="QWidget" name="FormJob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>С</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditBeg">
       <property name="displayFormat">
        <string>dd.MM.yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>По</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditEnd">
       <property name="displayFormat">
        <string>dd.MM.yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpd">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxZon">
       <property name="text">
        <string>Участок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DbComboBox" name="comboBoxZon">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRab">
       <property name="text">
        <string>Работник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DbComboBox" name="comboBoxRab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxPrim">
       <property name="text">
        <string>Примечание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdTruncatePrem">
       <property name="text">
        <string>Установить коэф. премии</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DbViewer" name="tableViewJob"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>db/dbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
